pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## Description
+Resolves #
+
```

Comment:
I love this template so much.  Very very very helpful and I'm stealing it for future work.